.github/workflows/cleanup-demo-images.yaml

```
@@ -108,32 +46,55 @@ jobs:
                   FILES_TO_REMOVE="$FILES_TO_REMOVE $file"
                   REMOVE_COUNT=$((REMOVE_COUNT + 1))
                 fi
-              fi
-            done
+              done
+            else
+              # Bulk cleanup - get open PRs and remove files for closed PRs
+              echo "🔍 Fetching open PR numbers..."
+              OPEN_PRS=$(gh pr list --json number --jq '.[].number' --limit 1000 | tr '\n' ' ')
+              echo "📋 Open PRs: $OPEN_PRS"
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)